src/Account.sol

```diff
@@ -257,10 +258,7 @@ contract Account is IAccount, Auth, OpsReady {
     /// @notice Decodes and executes the given command with the given inputs
     /// @param _command: The command type to execute
     /// @param _inputs: The inputs to execute the command with
-    function _dispatch(Command _command, bytes calldata _inputs)
-        internal
-        nonReentrant
-    {
+    function _dispatch(Command _command, bytes calldata _inputs) internal {
```

Comment:
Question: It doesn't need nonReentrant anymore because its internal?

Reply:
It just could be moved up a level to the `execute` function and it is still safe since `_dispatch` is internal and can *only* be called by execute.

"In `Account.sol`’s `dispatch()` function has a `nonReentrant` modifier that prevents reentrancy. However, `dispatch()` is internal and only called by `execute()` for each command. This means that the `locked` value will be read and set twice per call to dispatch for each command execute, but it only needs to be set once in `execute()`

**Remediations to Consider**

Move the `nonReentrant` modifier from `dispatch()` to the `execute()` function to prevent unnecessary calls to the `nonReentrant` modifier." -Macro